src/treemodel/schema.js

```diff
@@ -264,7 +300,28 @@ export default class Schema {
 		return this.checkQuery( {
 			name: name,
 			inside: Schema._makeItemsPathFromPosition( position ),
-			attribute: attribute
+			attributes: attributes
+		} );
+	}
+
+	/**
+	 * Checks whether entity with given name (and optionally, with given attribute(s)) is allowed in given chain of
+	 * parent {@link engine.treeModel.Element elements}.
+	 *
+	 * @param {Array.<engine.treeModel.Element>} elements Elements that are parents of queried entity.
+	 * @param {String} name Entity name to check.
+	 * @param {Array.<String>|String} [attributes] If set, schema will check for entity with given attribute(s).
+	 * @returns {Boolean} `true` if entity is allowed, `false` otherwise
+	 */
+	checkInElements( elements, name, attributes ) {
```

Comment:
The method name is missleading. It looks like the schema is checked for many elements. I do not think this method at all is needed, one can use `checkQuery`.